.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: "Release and Publish Package"
+
+on:
+  push:
+    branches:
+      - "main"
+
+jobs:
+  publish:
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+      - name: "Install or retrieve from cache node & npm."
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          registry-url: "https://registry.npmjs.org"
+
+      - name: Install
+        run: npm install --legacy-peer-deps
```

Comment:
what about using `npm ci` here? https://docs.npmjs.com/cli/v9/commands/npm-ci or also yarn alternative is `yarn install --immutable`

Reply:
fixed